src/Cli/UpCommand.php

```diff
@@ -42,8 +41,7 @@ protected function configure()
     {
         $this
             ->setName('up')
-            ->setDescription('Start the project')
-        ;
```

Comment:
Why removing that ? I'd say everything on the same line or everything on different lines.